packages/components/src/__future__/Tag/RemovableTag/RemovableTag.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react"
+import classNames from "classnames"
+import { Tag, TagProps } from "../Tag"
+import {
+  RemoveButton,
+  RemoveButtonBaseProps,
+} from "./subcomponents/RemoveButton"
+import styles from "./RemovableTag.module.scss"
+
+export type RemovableTagProps = {
+  removeButtonProps: RemoveButtonBaseProps
+} & Omit<TagProps, "color">
```

Comment:
Just double checking, should you omit `icon` here too? The [doc says no icons](https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/3324903621/RemovableTag).

Reply:
Cool, omitted & commented out like `color`